tests/test_klaviyo_automatic_fields.py

```diff
@@ -0,0 +1,74 @@
+from tap_tester import menagerie
+
+from tap_tester import connections, runner
+from base import KlaviyoBaseTest
+
+class AutomaticFieldsTest(KlaviyoBaseTest):
+    """
+    Ensure running the tap with all streams selected and all fields deselected results in the replication of just the 
+    primary keys and replication keys (automatic fields).
+    """
+    
+    def name(self):
+        return "klaviyo_automatic_fields_test"
+    
+    def test_run(self):
+        """
+        • Verify we can deselect all fields except when inclusion=automatic, which is handled by base.py methods
+        • Verify that only the automatic fields are sent to the target.
+        • Verify that all replicated records have unique primary key values.
+        """
+        expected_streams = self.expected_streams()
+        
+        conn_id = connections.ensure_connection(self)
+
+        # Run in check mode
+        found_catalogs = self.run_and_verify_check_mode(conn_id)
+        
+        # table and field selection
+        test_catalogs = [catalog for catalog in found_catalogs
+                                      if catalog.get('stream_name') in expected_streams]
+
+        # Select all streams and no fields within streams
+        self.perform_and_verify_table_and_field_selection(
+            conn_id, test_catalogs, select_all_fields=False)
+
+        record_count_by_stream = self.run_and_verify_sync(conn_id)
+        synced_records = runner.get_records_from_target_output()
+        
+        for stream in expected_streams:
+            
+            # WE ARE NOT ABLE TO GENERATE TEST DATA SO SKIPPING TWO STREAMS(mark_as_spam, dropped_email)
+            if stream in ["mark_as_spam", "dropped_email"]:
```

Comment:
Instead of this remove streams from the expected streams with comments. And if it is for all tests then remove this from base.py with comment.

Reply:
Updated